Sysvale/component-library
# feature/empty-state

#### What is the purpose of this pull request?
Creates the EmptyState component

#### How should this be manually tested?
* Test using the actions addon and the knobs addon;
* Try adding an image and alt text to the component;
* Try changing the text of the headline, body text and button;
* Try changing the color of the headline, body text and button;
* Test the maximum font size of the headline (60px), body text (30px) and button (30px);
* Try changing the border radius of the button;
* See if an action is triggered when the button is pressed;
* Verify if the predefined styles of the component passes the accessibility tests;

#### Screenshots or example usage
![image](https://user-images.githubusercontent.com/26309807/76973569-e1faf380-690e-11ea-8844-7eddc157e1f2.png)

#### Types of changes
